tests/conftest.py

```diff
@@ -277,6 +277,7 @@ def new_settings():
         pagination=Pagination(max_total_hits=17),
         separator_tokens=["&sep", "/", "|"],
         non_separator_tokens=["#", "@"],
+        search_cutoff_ms=None,
```

Comment:
Let's give this a value to test changing it.

```suggestion
        search_cutoff_ms=100
```